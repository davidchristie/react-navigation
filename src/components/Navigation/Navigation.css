@media screen and (max-width: 400px) {
  .Navigation.collapsed ul {
    display: none;
  }
}
