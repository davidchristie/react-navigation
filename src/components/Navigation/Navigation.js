import React, { Component } from 'react'

export default class Navigation extends Component {
  render () {
    return (
      <nav className='Navigation'>
        <ul>
          {this.props.children}
        </ul>
      </nav>
    )
  }
}
