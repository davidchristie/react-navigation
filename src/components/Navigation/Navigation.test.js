import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import React from 'react'

import Navigation from './Navigation'

it('renders correctly', () => {
  expect(toJson(shallow(
    <Navigation>
      Content
    </Navigation>
  ))).toMatchSnapshot()
})
