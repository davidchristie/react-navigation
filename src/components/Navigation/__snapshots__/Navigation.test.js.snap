// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<nav
  className="Navigation"
>
  <ul>
    Content
  </ul>
</nav>
`;
